monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunMonkeyPage.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {Col} from 'react-bootstrap';
+import {Link} from 'react-router-dom';
+import AuthComponent from '../../AuthComponent';
+import RunOptions from './RunOptions';
+
+
+class RunMonkeyPageComponent extends AuthComponent {
+
+  render() {
+    return (
+      <Col sm={{offset: 3, span: 9}} md={{offset: 3, span: 9}}
+           lg={{offset: 3, span: 9}} xl={{offset: 2, span: 7}}
```

Comment:
This col should be span 12 in all sizes probably

Reply:
I did not change this part, so it looks as it did before